scripts/lua/host_details.lua

```diff
@@ -687,6 +687,45 @@ end
    <button class="btn btn-default" onclick="$('#reset_host_stats_dialog').modal('show')">]] print(i18n("host_details.reset_host_stats")) print[[</button>
    </td></tr>]]
 
+
+   --==========================WIP=======================================
```

Comment:
is this WIP expected? do you still have to check if the arp matrix is enabled? In case it is very easy, you can simply read boolean `is_arp_matrix_generation_enabled` from the prefs in lua.